bin/node/cli/src/cli.rs

```diff
@@ -47,8 +47,8 @@ pub enum Subcommand {
 	#[structopt(name = "benchmark", about = "Benchmark runtime pallets.")]
 	Benchmark(frame_benchmarking_cli::BenchmarkCmd),
 
-	/// Try some experimental command on the runtime. This includes migration and runtime-upgrade
-	/// testing.
+	/// Try some experimental command on the runtime. See `try-runtime --help` for more.
+	/// NOTE: `try-runtime` feature must be enabled.
```

Comment:
> It may be worth adding something like

This still won't show up unless `try-runtime` is enabled, correct?

> an executor for that subcommand

I am not sure what this means - any chance you can elaborate or link docs/example?

Reply:
> This still won't show up unless try-runtime is enabled

No, I was suggesting an alternate implementation which is only compiled when `try-runtime` is _not_ enabled.

> not sure what this means

[this pattern](https://github.com/coriolinus/calc/blob/ec5e430f877cf7784b422fdf417ed9532cf26066/src/main.rs#L1-L10): a tiny stub which just gives a nice human-readable error if you don't have the compile flag set, so that the CLI interface remains the same whether or not the feature is enabled.

Reply:
the stub sounds like a good idea to me.